docs/resources/secret_folder.md

```diff
@@ -0,0 +1,33 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "oktapam_secret_folder Resource - terraform-provider-oktapam"
+subcategory: ""
+description: |-
+  A file-system like construct that contains secrets or nested secret folders. For details, see Secret Folders https://help.okta.com/okta_help.htm?type=oie&id=ext-pam-secrets-folder
+---
+
+# oktapam_secret_folder (Resource)
+
+A file-system like construct that contains secrets or nested secret folders. For details, see [Secret Folders](https://help.okta.com/okta_help.htm?type=oie&id=ext-pam-secrets-folder)
```

Comment:
This link also just takes me to generic OIE docs

Reply:
@isaacdurham-okta informed me that the doc links will be live next week, but we should be using this format